### ISSUE TITLE
Installer does not accuse of missing php Soap extension

### ISSUE THREAD

**Issue by gabrielqs** [magento/magento2#9587]
### Preconditions

1. Magento 2.1.6
2. Amazon Linux
3. PHP 7.0.16
4. Amazon RDS (MariaDB)

The server was completely installed using yum, with the packages available through the official Amazon repository

### Steps to reproduce

1. Create a new Amazon linux image on AWS
2. Install php, nginx, the whole thing except the php soap extension
3. Create a new vhost, copy over the 2.1.6 release
4. Composer install
5. Install magento through the command line ./bin/magento

### Expected result

The installation should not proceed as the soap extension, required for the magento soap api to work, is not present

### Actual result

The installation completes successfully, but the SOAP api is unusable

This is probably platform independent.

**Comment by TomashKhamlai** [comment]
Hello! Thank you for your submission. I will try to reproduce this issue. In the 4-th step, after I cloned Magento from Github, the error appears. Please look: 

```text
$ composer install
Cannot load Xdebug - it was already loaded
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested PHP extension ext-soap * is missing from your system. Install or enable PHP's soap extension.
```
Is this problem actual for you?

**Comment by gabrielqs-redstage** [comment]
That did not appear to me, not sure why... did you use amazon linux?

**Comment by TomashKhamlai** [comment]
No, I did not. The only way that I was able to install Magento without soap extension through Command Line Interface or Web Setup Wizard is going by this scenario.
1. Prepare everything that will be used for installation. (folder, vhost...)
2. Remove SOAP extension from system (`sudo yum erase php70w-soap -y`). But do not restart your webserver!
3. Install without SOAP. Restart your webserver.

I will try again later.
